aQute.libg/src/aQute/libg/runtime/JavaRuntime.java

```diff
@@ -0,0 +1,42 @@
+package aQute.libg.runtime;
+
+import java.io.File;
+import java.lang.management.ManagementFactory;
+import java.lang.management.RuntimeMXBean;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+public class JavaRuntime {
```

Comment:
This class now requires java.lang.management for the launcher. This may or may not be an issue for some users. We have tried to make sure the launcher side runs on older systems. This is why, for example, we limit OSGi to 4.2 and not later. I suspect that some older systems may have issues with the JMX usages in the launcher. Perhaps @pkriens can comment on his customer's use.